packages/search-ui/src/SearchDriver.js

```diff
@@ -33,8 +35,9 @@ export const DEFAULT_STATE = {
   filters: [],
   resultsPerPage: 20,
   searchTerm: "",
-  sortDirection: "",
-  sortField: "",
+  sortDirection: "asc",
+  sortField: "title",
```

Comment:
Oof, I completely missed this on my first pass. This definitely shouldn't be part of the DEFAULT_STATE.

```suggestion
  sortDirection: "",
  sortField: "",
```

Reply:
Woops, didn't even realize this was added in this PR, my mistake

Reply:
Resolved by https://github.com/elastic/search-ui/pull/592/commits/5f695d858496dec2e99503a70cb4c8036a3bd805